build/stf-run-ci/tasks/setup_base.yml

```diff
@@ -103,13 +103,13 @@
       apiVersion: operators.coreos.com/v1alpha1
       kind: Subscription
       metadata:
-        name: prometheus
+        name: rhods-prometheus-operator
         namespace: "{{ namespace }}"
       spec:
         channel: beta
```

Comment:
Is this the right channel?

Reply:
Yep, channel remained the same (I did look for it during docs updates)